tools/src/generator.ts

```diff
@@ -196,10 +196,14 @@ function mapOption(prop: IProp): IOption {
     } : {
       name: prop.name,
       isSubscribable: prop.isSubscribable || undefined,
-      nested: prop.props.map(mapOption)
+      nested: prop.props.map(mapOption),
+      isArray: isNestedOptionArray(prop)
     };
 }
-
+function isNestedOptionArray(prop: IProp): boolean {
+  const type = prop?.types?.[0]?.type;
+  return type ? type === "Array" : false;
```

Comment:
As I see, the `prop` is always exists. We can replace `prop?.` => `prop.`

Also, in `helpers` we have a function `isNotEmptyArray`. So I suggest that we can replace this code by:
```ts
return isNotEmptyArray(prop.types) && prop.types[0].type;
```